docs/design/automatic-updates.md

```diff
@@ -0,0 +1,38 @@
+
+# Storage Node automatic updates DesignDocument
+
+## Overview
+
+As more Storage Node Operators join the network we must ensure these nodes have a mechanism to automatically keep their nodes software up to date. If a Storage Node Operator fails to keep their node up to date with the minimum version required by satellites they will no longer be selected for upload or download requests.
+
+## Goals
+
+Create an automatic update system into the Storage Node software so that node operators do not have to worry about updating their nodes.
+Keep the Storage Nodes on the network up to date with the latest software to improve stability and performance. 
+Concerns
+
+A majority or substantial amount of nodes update their software at the same time causing instability in the network or temporary file lose.
+We must ensure the automatic update system we build into the Storage nodes works on ALL OS we support.
```

Comment:
Which OS's are those?  If there is a concrete list, we should link to it or duplicate it here.  If there's not, we need to create one here.

Reply:
@iglesiasbrandon Wrote that part